var/spack/repos/builtin/packages/geopm/package.py

```diff
@@ -45,24 +36,20 @@ class Geopm(AutotoolsPackage):
     variant('doc', default=True, description='Create man pages with ruby-ronn.')
     variant('openmp', default=True, description='Build with OpenMP.')
     variant('ompt', default=False, description='Use OpenMP Tools Interface.')
-    variant('hwloc', default=False, description='Build with hwloc, deprecated and ignored after v0.5.1.')
     variant('gnu-ld', default=False, description='Assume C compiler uses gnu-ld.')
 
     # Added dependencies.
-    depends_on('m4', type='build')
-    depends_on('automake', type='build')
-    depends_on('autoconf', type='build')
-    depends_on('libtool', type='build')
     depends_on('ruby-ronn', type='build', when='+doc')
-    depends_on('doxygen', type='build', when='+doc')
```

Comment:
There are a few dependencies that have been removed: `numatcl`, `doxygen`, `json-c` and `hwloc`. Are they not necessary anymore? How is documentation built if the package doesn't use `doxygen`?

Reply:
These are dependencies which were removed and are also no offered in the ./configure step anymore.
numactl and hwloc was replaced by linux functionality, json-c is shipped as contributing code, ronn is used for man-page generation and documentation.